scripts/network-launch/cluster-config/dev.txt

```diff
@@ -0,0 +1,12 @@
+bootNodeIP="164.90.173.129"
```

Comment:
Rename folder `cluster-config` to `clusters`.

Reply:
Rename `.txt` everywhere. Just `dev`, `dev1`, etc.

Reply:
updated.